0000-pr-merge-requirements.md

```diff
@@ -0,0 +1,55 @@
+- Feature Name: PR Merge Requirements
+- Start Date: 2019-05-21
+- SEP Status: Draft
+- SEP PR: (leave this empty)
+- Salt Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Lay out the requirements to merge a PR
+
+# Motivation
+[motivation]: #motivation
+
+As we begin our working groups, to make sure that our community is all on the same page, we want to
+clearly define our requirements to merge a PR. This should benefit developers, reviewers and those
+merging the PRs.
+
+# Design
+[design]: #detailed-design
+
+All PR requirements
+  - Approval Reviews: 1 approval review from core team member OR
+                      1 approval review from captain of working group
+  - All tests pass.
+  - Cannot merge your own PR until 1 reviewer approves from defined list above that is not the author.
+
+Bug Fix PR requirements:
+  - Test Coverage: regression test written to cover bug fix.
```

Comment:
Do makes sense to create always an github issue and reference the issue number in the comment?

This will eventually make easy to track the status of the bugs.

Reply:
i like that idea :)